README.md

```diff
@@ -1,15 +1,25 @@
-Exchange Calendar is an **add-on for Thunderbird and Lightning** that provides **Exchange calendar support**.
+Exchange EWS Provider is an **add-on for
+[Thunderbird](https://mozilla.org/thunderbird) and
+[Lightning](https://mozilla.org/calendar)** that provides
+**[Microsoft Exchange](http://microsoft.com/exchange) calendar
+support**.
 
 ## Contributing
-This add-on is open-source and based on the work of many [contributors](https://github.com/ExchangeCalendar/exchangecalendar/graphs/contributors).
 
-### Installing developement tools
-You will need to install Dom Inspector, Javascript Debugger and some other development tools. Fortunately `Thunderbird addons` makes it easy to install all of these: 
-  
-Now using bash we'll compile the addon we need (which are written in wiki)
+This add-on is open source and based on the work of many
+[contributors](https://github.com/ExchangeCalendar/exchangecalendar/graphs/contributors).
+
+### Installing the developement tools
 
-    cd ./exchangecalendar_master; chmod +x ./build.sh; ./build.sh;
+You will need to install the
+[DOM Inspector](https://addons.mozilla.org/en-US/thunderbird/addon/dom-inspector-6622/),
+the
+[JavaScript Debugger](https://addons.mozilla.org/thunderbird/addon/javascript-debugger/),
```

Comment:
Since these tools no longer seem to be necessary, please remove the entire paragraph.